### ISSUE TITLE
Docs Improvement Suggestion: Tween.follow_property()

### ISSUE THREAD

**Issue by metcalf-putnam** [godotengine/godot-docs#5693]
**Your Godot version:** 
3.4.3

**Issue description:**
Docs clarification needed for Tween's follow_property() method. As described, it seems to imply target and object functionalities are swapped from actual implementation. Similar treatment may be needed for follow_method().

_Current:_
`Follows property of object and applies it on target_property of target, beginning from initial_val for duration seconds, delay seconds later.`

Current wording implies that the object's property is applied to the target, that is, that we are animating the target instead of the object. The implementation of the method animates the object based on the target, not the other way around.

_Suggested:_
`Animates property of object, beginning from initial_val, and follows target_property of target as its ending value, for duration seconds, delay seconds later.`

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/classes/class_tween.html#class-tween-method-follow-property

**Comment by stephannv** [comment]
Has the `follow_property` been removed from new Tween (Tween in Godot 4 and SceneTreeTween)?

**Comment by Vennnot** [comment]
Yes it has. Clarification could still be there for the 3.x documentation. We need a label to be able to distinguish between 3.x documentation issues and newer ones. If we're not maintaining it, then this issue can be closed.
